spec/classes/plugin/dns_cloudflare.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'letsencrypt::plugin::dns_cloudflare' do
+  on_supported_os.each do |os, facts|
+    context "on #{os} based operating systems" do
+      let(:facts) { facts }
```

Comment:
It is now recommended to have a different name for the `let` block and the regular Ruby variable:
```suggestion
  on_supported_os.each do |os, os_facts|
    context "on #{os} based operating systems" do
      let(:facts) { os_facts }
```

Reply:
👍